src/configs/beta.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* @ts-ignore */
+import { baseConfig, chooseUrl } from "./configs/base";
```

Comment:
We can avoid this by using absolute imports

```suggestion
import { baseConfig, chooseUrl } from "src/configs/base";
```

Reply:
@michael1011 added it, also had to define this path on `vite.config.mjs`:
```
    resolve: {
        alias: {
            src: path.resolve(__dirname, "src"),
        },
    },
```
Opted to define just this one as we usually don't use aliases.